Jobs/test/Job.IdentityDeletion.Tests/Tests/ActualDeletionWorkerTests.cs

```diff
@@ -41,24 +44,28 @@ public async Task Proxies_triggering_the_deletion_processes_to_command_handler()
     public async Task Calls_Deleters_For_Each_Identity()
     {
         // Arrange
-        var mediator = A.Fake<IMediator>();
+        var mockMediator = A.Fake<IMediator>();
         var identityAddress1 = TestDataGenerator.CreateRandomIdentityAddress();
         var identityAddress2 = TestDataGenerator.CreateRandomIdentityAddress();
-        SetupRipeDeletionProcessesCommand(mediator, identityAddress1, identityAddress2);
+        SetupRipeDeletionProcessesCommand(mockMediator, identityAddress1, identityAddress2);
+
         var mockIdentityDeleter = A.Fake<IIdentityDeleter>();
-        var identityDeleters = new List<IIdentityDeleter>([mockIdentityDeleter]);
+        var dummyDevicesIdentityDeleter = new IdentityDeleter(mockMediator);
```

Comment:
I think you should test calls to this as well. Previously all IIdentityDeleters where in a single list, now that you separated them the target of the test should not change.

Reply:
Maybe I'm mistaken or missing something but in this line `var identityDeleters = new List<IIdentityDeleter> { mockIdentityDeleter, dummyDevicesIdentityDeleter };` we are puting them in one list and passing to `CreateWorker`. I tried to setup the separate call but didn't managed it. I debugged the code and saw that the `ActualDeletionWorker` sucessfully separates `DevicesIdentityDeleter` from other. So I would not change the test.